### PR TITLE
Fix issue 168 Macro's using `(__extension__({}))`

### LINKED ISSUE #168: Macro's using "(__extension__ ({ ... }))" generate 'Untranslatable cursor'
d++ generates this warning when it encounters macro's that use the `__extention__` flag. They mostly look like:
```c
#define MACRO(X) (__extension__({ ... }))
```
Example Header files using this `__extension__`:
- /usr/include/netinet/in.h
- /usr/include/x86_64-linux-gnu/bits/cpu-set.h

Simple Solution (unsure if this is a good/bad idea):
```diff
diff --git a/source/dpp/translation/translation.d b/source/dpp/translation/translation.d
index 361e8cc..302712c 100644
--- a/source/dpp/translation/translation.d
+++ b/source/dpp/translation/translation.d
@@ -257,7 +257,7 @@ bool untranslatable(in string line) @safe pure {
         || line.canFind("variant!")
         || line.canFind("value _ ")
         || line.canFind("enable_if_c")
-        || line.canFind(`}))`)  // ???
+//        || line.canFind(`}))`)  // ???
         || line.canFind(`(this_)_M_t._M_equal_range_tr(`)
         || line.canFind(`this-`)
         || line.canFind("_BoundArgs...")
```

It could be that this simple solution is just ok, or it might be that testing for the last line, without looking at the beginning is not 100% conclusive.
I am currently trying to figure out where to add an example test so that i can attach a PR for the issue.

P.S. Thanks for your talk at DConf2019 about d++ bringing it to my attention again.

### PR DESCRIPTION
**Pull request by dkgroot**
Includes a test for the issue and a simple 'fix'. Not sure if the fix is the way you would like to go though.
references #168
fixes #168

### CODE REVIEW

File: source/dpp/translation/translation.d [hunk 1]
```diff
@@ -257,7 +257,8 @@ bool untranslatable(in string line) @safe pure {
         || line.canFind("variant!")
         || line.canFind("value _ ")
         || line.canFind("enable_if_c")
-        || line.canFind(`}))`)  // ???
+        // causing trouble with gcc __extension__ (#168)
+        // || line.canFind(`}))`)  // ???
```
Comment (atilaneves) [inline source/dpp/translation/translation.d]: If this can be deleted without breaking anything, please do that instead of commenting it out.

Comment (dkgroot) [reply source/dpp/translation/translation.d]: Updated ldc2 to 1.15... Retesting.

Comment (dkgroot) [reply source/dpp/translation/translation.d]: All tests passed :-)

**Comment by dkgroot** [comment]
Thanks Atila

**Pull request merged**